src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,78 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  todos: Todo[];
+  selectedTodoId: number;
+  setSelectedTodoId: (id: number) => void;
+};
 
-    <tbody>
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
+export const TodoList: React.FC<Props> = ({
+  todos,
+  selectedTodoId: selectedTodoId,
```

Comment:
You don't need to rename the destructured properties if the names are the same. So instead of `selectedTodoId: selectedTodoId`, you can simply write `selectedTodoId`.